static/src/javascripts/projects/common/modules/analytics/mvt-cookie.js

```diff
@@ -6,10 +6,7 @@ define([
     var MULTIVARIATE_ID_COOKIE = 'GU_mvt_id',
         VISITOR_ID_COOKIE = 's_vi',
         BROWSER_ID_COOKIE = 'bwid',
-        // The full mvt ID interval is [1, 1000000]
-        // Server side mvt IDs occupy [9000000, 1000000)
-        // So the client-side mvt interval is [1, 899999]
-        MAX_CLIENT_MVT_ID = 899999;
+        MAX_CLIENT_MVT_ID = 1000000; // The full mvt ID interval is [1, 1000000]
```

Comment:
Yes, this will increase the size of each allocation, and people in one set of tests could be removed or pushed across to another test.